src/utf8.rs

```diff
@@ -43,7 +43,8 @@ const ACCEPT: usize = 12;
 const REJECT: usize = 0;
 
 #[cfg_attr(rustfmt, rustfmt::skip)]
-static CLASSES: [u8; 256] = [
+//SAFETY: unsafe code in the decode function rely on the correctness of this state machine.
```

Comment:
Could you make this comment (and the ones below) are formatted in a way that is consistent with other comments? Namely:

* Lines should be no longer than 79 columns, inclusive.
* Comments on items like this should use three slashes.
* There should be a space after the `///` and before the first letter.

i.e., `/// Foo` and not `//Foo`.